modules/openapi-generator/src/main/resources/python/__init__.mustache

```diff
@@ -0,0 +1,13 @@
+# coding: utf-8
```

Comment:
Can you change this file name to __init__test.mustache
to follow the naming convention that we use [here](https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator/src/main/resources/python)?

Reply:
Ok, I will do that, but I will keep a copy of `__init__.mustache` for supporting files (see: `supportingFiles.add(new SupportingFile("__init__.mustache", currentPackagePath, "__init__.py"));`)

Reply:
I have the new __init__test.mustache, but I am leaving the __init__.mustache, used for packages. Look at https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java after the comment "// If the package name consists of dots(openapi.client), then we need to create the directory structure like openapi/client with __init__ files."

Nothing new on this respect.

Reply:
Ah, I understand that we need this file now. Thank you for explaining that.
Why do we need this file content changed though?
It looks like it worked as-s when it was empty before. Can you revert it to being empty?

Reply:
You are right. I changed it for the tests, then changed back for the packages, but I reverted it badly.
I will revert it.